daemon/dns/dns.go

```diff
@@ -73,7 +81,8 @@ func (d *DefaultSetter) Set(iface string, nameservers []string) error {
 		if method.IsAvailable() {
 			d.publisher.Publish("set dns for interface [" + iface + "] using: " + method.Name())
 			if err := method.Set(iface, nameservers); err != nil {
-				return fmt.Errorf("setting dns with %s: %w", method.Name(), err)
+				log.Println(internal.ErrorPrefix, fmt.Errorf("setting dns with %s: %w", method.Name(), err))
```

Comment:
Shouldn't the used option be marked and use the same one for disconnect? And maybe if that fails loop thru the list with all the commands?
Otherwise wouldn't there be issues if the user changes the system while connected to VPN and at disconnect something else is used?

Reply:
I don't think it is necessary, because DNS settings are per-interface in most modern linux distros, and we do created/destroy our virtual interface, or set/reset DNS settings for interface every time when it is needed. Also, it is no problem to set DNS with one method, and unset with another. Except direct file resolv.conf modify which is the only method in docker container.

Reply:
yeah, I was thinking on that with the conf file. because the others the OS would take care of it when the interface is gone.